components/formats-bsd/src/loci/formats/in/MicromanagerReader.java

```diff
@@ -611,21 +611,21 @@ private void parseKeyAndValue(String key, String value, int digits, int plane, i
       if (key.equals("XPositionUm")) {
         try {
           Position p = positions.get(getCoreIndex());
-          p.positions[i][0] = new Double(value);
+          p.positions[i][0] = DataTools.parseDouble(value);
         }
         catch (NumberFormatException e) { }
       }
       else if (key.equals("YPositionUm")) {
         try {
           Position p = positions.get(getCoreIndex());
-          p.positions[i][1] = new Double(value);
+          p.positions[i][1] = DataTools.parseDouble(value);
         }
         catch (NumberFormatException e) { }
```

Comment:
With the usage of the `DataTools` API, could we further simplify this code and remove the `NumberFormatException` handling in the reader?

Reply:
Done in 59ed134